themes/digital.gov/src/scss/new/_uswds-theme.scss

```
@@ -45,8 +44,8 @@ i.e.
   $theme-body-line-height: 4,
   $theme-style-body-element: true,
   $theme-text-measure: 5,
-  //Colors from _uswds_theme-color.scss
-  $theme-color-base-lightest: "gray-cool-5",
+  /*Colors from _uswds_theme-color.scss*/ $theme-color-base-lightest:
```

Let's return to the SCSS comment.
```suggestion
  // Colors from _uswds_theme-color.scss
  $theme-color-base-lightest:
```